cli/cmd/lql_create.go

```diff
@@ -0,0 +1,69 @@
+//
+// Author:: Salim Afiune Maya (<afiune@lacework.net>)
+// Copyright:: Copyright 2020, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// lqlCreateCmd represents the lql create command
+	lqlCreateCmd = &cobra.Command{
+		Use:   "create",
+		Short: "create an LQL query",
+		Long:  `Create an LQL query.`,
+		Args:  cobra.NoArgs,
+		RunE:  createQuery,
+	}
+)
+
+func init() {
+	// add sub-commands to the lql command
+	lqlCmd.AddCommand(lqlCreateCmd)
+
+	setQueryFlags(lqlCreateCmd.Flags())
+}
+
+func createQuery(cmd *cobra.Command, args []string) error {
+	lqlCreateUnableMsg := "unable to create LQL query"
```

Comment:
This is used in a wrapped manner, so there should be no confusion as it will be accompanied by a more specific error.  It's a variable simply to save on duplicate strings, but we can easily explode it.

Reply:
Fixed.